app/Helpers/database/set-claim.php

```diff
@@ -182,7 +182,16 @@ function getClaimData(int $gameID, bool $getFullData = true): array
         $query .= diffMinutesPassedStatement('sc.Created', 'MinutesActive');
     }
 
-    $query .= " FROM SetClaim sc WHERE sc.GameID = '$gameID'
+    if (is_array($gameID)) {
+        if (empty($gameID)) {
+            return [];
+        }
+        $gameIDs = implode(',', $gameID);
+    } else {
+        $gameIDs = $gameID;
+    }
+
+    $query .= " FROM SetClaim sc WHERE sc.GameID IN ($gameIDs)
```

Comment:
Fetching 25 claims in one query took about as long as fetching 2 claims individually.